mkdocs.yml

```diff
@@ -96,57 +107,49 @@ nav:
       - ReferenceGrant: api-types/referencegrant.md
     - API specification: reference/spec.md
     - Policy Attachment: reference/policy-attachment.md
-    - Enhancement Proposals:
-      - Overview: geps/overview.md
-      - Memorandum:
-        - geps/gep-917/index.md
-        - geps/gep-922/index.md
-        - geps/gep-1324/index.md
-        - geps/gep-2659/index.md
-      - Declined:
-        - geps/gep-735/index.md
-        - geps/gep-1282/index.md
-      - Provisional:
-        - geps/gep-91/index.md
-        - geps/gep-1619/index.md
-        - geps/gep-1651/index.md
-        - geps/gep-1867/index.md
-      # - Implementable:
-      #   -
-      - Experimental:
-        - geps/gep-713/index.md
-        - geps/gep-957/index.md
-        - geps/gep-1016/index.md
-        - geps/gep-1426/index.md
-        - geps/gep-1686/index.md
-        - geps/gep-1709/index.md
-        - geps/gep-1742/index.md
-        - geps/gep-1748/index.md
-        - geps/gep-1762/index.md
-        - geps/gep-1897/index.md
-        - geps/gep-1911/index.md
-        - geps/gep-2162/index.md
-        - geps/gep-2257/index.md
-      - Standard:
-        - geps/gep-709/index.md
-        - geps/gep-718/index.md
-        - geps/gep-724/index.md
-        - geps/gep-726/index.md
-        - geps/gep-746/index.md
-        - geps/gep-820/index.md
-        - geps/gep-851/index.md
-        - geps/gep-1323/index.md
-        - geps/gep-1364/index.md
+  - Enhancements:
```

Comment:
Does it make more sense to have this section sorted in roughly _declining_ order of stability? I'm thinking:

1. Memorandum
2. Standard
3. Experimental
4. Implementable
5. Provisional

Reply:
Since this PR is big enough to conflict with ~everything docs related, I'm going to merge as is and make a follow up for this detail so I don't have to do any more rebases.